### ISSUE TITLE
request support for SMTP server that cannot enable TLS

### ISSUE THREAD

**Issue by evanlabs** [Leantime/leantime#134]
If configure.php can disable TLS with $smtpAutoTLS = false, it will be more friendly to SMTP servers that cannot enable TLS.

Add a line of code to the class mailer constructor of "class.mailer.php"
`$this->mailAgent->SMTPAutoTLS = $config->smtpAutoTLS;`

For security and compatibility, $smtpAutoTLS in configuration.sample.php defaults to true.
`public $smtpAutoTLS = true;`

**Comment by evanlabs** [comment]
Sorry, my pull request is for reference only.

**Comment by marcelfolaron** [comment]
Thank you! I can see how that would be a possible, scenario. Will add that.

**Comment by marcelfolaron** [comment]
I'll dive a little deeper into PHPMailer and see how the relationship between autotls and smtpsecure works. What happens if autoTLS is true and secure is set to ssl?

**Comment by evanlabs** [comment]
There is no support for secure="ssl" in PHPMailer, even if SMTPAutoTLS=true

https://github.com/PHPMailer/PHPMailer/blob/master/src/PHPMailer.php
```
if ($this->smtp->connect($prefix . $host, $port, $this->Timeout, $options)) {
    try {
        if ($this->Helo) {
            $hello = $this->Helo;
        } else {
            $hello = $this->serverHostname();
        }
        $this->smtp->hello($hello);
        //Automatically enable TLS encryption if:
        // * it's not disabled
        // * we have openssl extension
        // * we are not already using SSL
        // * the server offers STARTTLS
        if ($this->SMTPAutoTLS && $sslext && 'ssl' !== $secure && $this->smtp->getServerExt('STARTTLS')) {
            $tls = true;
        }
        if ($tls) {
            if (!$this->smtp->startTLS()) {
                throw new Exception($this->lang('connect_host'));
            }
            // We must resend EHLO after TLS negotiation
            $this->smtp->hello($hello);
        }
        if ($this->SMTPAuth && !$this->smtp->authenticate(
            $this->Username,
            $this->Password,
            $this->AuthType,
            $this->oauth
        )) {
            throw new Exception($this->lang('authenticate'));
        }

        return true;
    } catch (Exception $exc) {
        $lastexception = $exc;
        $this->edebug($exc->getMessage());
        // We must have connected, but then failed TLS or Auth, so close connection nicely
        $this->smtp->quit();
    }
}
```

**Comment by marcelfolaron** [comment]
Included in 2.0.13
